docs/architecture/gas/parameter_definition.md

```diff
@@ -6,8 +6,61 @@ IMPORTANT: This is not the final list of parameters, it contains the base values
 which can be overwritten per protocol version. For example,
 [53.txt](core/primitives/res/runtime_configs/53.txt) changes several parameters
 starting from version 53. To see all parameter values for a specific version at
-once, check out list of JSON snapshots generated in this directory:
+once, check out the list of JSON snapshots generated in this directory:
 [core/primitives/src/runtime/snapshots](https://github.com/near/nearcore/blob/master/core/primitives/src/runtime/snapshots).
 
-<!-- TODO: How to add a new parameter -->
 <!-- TODO: Runtime Config Store -->
+
+## How to Add a New Parameter
+
```

Comment:
Let's add "if you are feeling lost, open a thread at zulip/contract-runtime, we are here to help you"

Reply:
Good idea